### ISSUE TITLE
integral of secant shouldn't need principal value

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3761]
Imported from SourceForge on 2024-07-08 14:56:11
Created by **[kcrisman](https://sourceforge.net/u/kcrisman/)** on 2015-01-09 02:38:55
Original: https://sourceforge.net/p/maxima/bugs/2880

---

~~~~
(%i1) integral(sec(x), x, -%pi/4, %pi/4);
                                              %pi  %pi
(%o1)                   integral(sec(x), x, - ---, ---)
                                               4    4
(%i2) integrate(sec(x), x, -%pi/4, %pi/4);
Principal Value
                         sqrt(2) + 2          sqrt(2) - 2
(%o2)                log(-----------) - log(- -----------)
                              2                    2
~~~~

But a principal value shouldn't be necessary for this.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 14:56:13
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2018-05-23 16:23:17
Original: https://sourceforge.net/p/maxima/bugs/2880/#920a

---

Fixed by [82491a4d4956940aff69192406d601aecf14eb12]

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 14:56:16
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2018-05-23 16:23:35
Original: https://sourceforge.net/p/maxima/bugs/2880/#8fc4

---

- **status**: open --> closed
